src/components/ebay-fake-menu/index.marko

```diff
@@ -53,6 +54,8 @@ $ var baseClass = input.classPrefix || "fake-menu";
                         ...processHtmlAttributes(item, itemIgnoredAttributes)
                         class=[`${baseClass}__item`, item.class]
                         style=item.style
+                        disabled=(item.type === "button" && item.disabled)
+                        aria-disabled=(item.type !== "button" && item.disabled ? 'true' : null)
```

Comment:
Just for consistency, doesn't this work?:
`aria-disabled=(item.type !== "button" && item.disabled && 'true')`
No readon to default to null I'm assuming